subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_pgps

```diff
@@ -93,4 +93,26 @@ config NRF_CLOUD_PGPS_SOCKET_RETRIES
 	help
 	  This sets the maximum number of times to retry a download.
 
+config NRF_CLOUD_PGPS_PARTITION
+	bool "Define P-GPS subsystem partition"
+	default y if !MEMFAULT && SECURE_BOOT
```

Comment:
Just curious, why does `SECURE_BOOT` affect whether or not this partition should exist?

Reply:
Perhaps `SECURE_BOOT` is not the correct symbol to use, but it is interrelated.  If it is enabled, that implies mcuboot loader FOTA updates are enabled, which cause conflicts with P-GPS when the mcuboot secondary partition is used for both.  Basically, the mcuboot secondary partition is used to store the downloaded image, and that image must remain through a number of reboot loops before the mcuboot update is fully reported to the cloud and confirmed by the primary bootloader.  But P-GPS will notice its flash data is corrupt, and start downloading new data overtop of the image still required in the secondary area.

If you have a better idea, please let me know.